src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -303,36 +318,30 @@ class Bots extends Component {
   }
 
   renderDetails() {
+    const { categories, category, description, error, name, status } = this.state
     return (
       <div>
-        {this.state.error && <Alert color="danger">{this.state.error.message}</Alert>}
-        {this.state.successMsg && <Alert type="success">{this.state.successMsg}</Alert>}
+        {error && <Alert color="danger">{error.message}</Alert>}
```

Comment:
You replaced success alert with the `toastSuccess` util, why not use `toasError` with errors ?

Reply:
I thought that an error should maybe be persistent, especially if it's related to a value in the form that needs to be changed, but it might not make sense in this context, I don't know the form enough to know all the possible errors that may be displayed. Maybe it could be a `toastFailure` saying that the form contains errors with error messages on the fields that need to be changed, but I didn't go far down in the errors

Reply:
I added a toast message so the user can have visual feedback if there's an error and they don't have the Alert at the top of their form in the viewport. I left the Alert to provide more info as to how they can solve the error. I also fixed an issue I had not noticed that the `isSaving` state would stay active even when there was an error leaving the form save and cancel disabled forever